### ISSUE TITLE
Possibility of changing datapoints in the program section.

### ISSUE THREAD

**Issue by Ji-Xinyou** [opencsapp/opencsapp.github.io#171]
New applicants are mostly updating their datapoints in the datapoints section, the datapoints in the program section are not updated for a while. I think the datapoints in the program section could be simply removed or updated as a hyperlink to the datapoints section, otherwise it will be stale in a few years.

**Comment by xichenpan** [comment]
That's true, we are working on providing a link to ads/rejs datapoints in each program page.

**Comment by Ji-Xinyou** [comment]
> That's true, we are working on providing a link to ads/rejs datapoints in each program page.

I would close this since it has been WIP.
